content/docs/1.6.0/advanced-resources/os-distro-specific/okd-support.md

```diff
@@ -0,0 +1,164 @@
+---
+title:  OCP/OKD Support
+weight: 4
+---
+
+To deploy Longhorn on a cluster provisioned with OpenShift 4.x, some additional configurations is required.
+
+- [Prerequisite](#prerequisite)
+  - [Default Path /var/lib/longhorn Setup](#default-path-varliblonghorn-setup)
+  - [Separate Path /var/mnt/longhorn Setup](#separate-path-varmntlonghorn-setup)
+    - [Create Filesystem](#create-filesystem)
+    - [Mounting Devices On Boot with MachineConfig](#mounting-devices-on-boot-with-machineconfig)
+    - [Label and Annotate Nodes](#label-and-annotate-nodes)
+- [Install Longhorn](#install-longhorn)
+- [Upgrade Longhorn](#upgrade-longhorn)
+- [Uninstall Longhorn](#uninstall-longhorn)
+  - [Uninstall Prerequisite](#uninstall-prerequisite)
+  - [Uninstalling Longhorn using kubectl](#uninstalling-longhorn-using-kubectl)
+- [Reference](#reference)
+
+## Prerequisite
+
+### Default Path /var/lib/longhorn Setup
+
+Label each node for storage with:
+
+```bash
+oc get nodes --no-headers | awk '{print $1}'
+oc label node "${NODE_NAME}" node.longhorn.io/create-default-disk=true
+```
```

Comment:
https://longhorn.io/docs/1.5.1/references/settings/#create-default-disk-on-labeled-nodes is disabled by default, so that means the default disk will be created automatically, so do we still need this step? or actually users just follow the general process for disk configuration.

Reply:
I think the docs is missing the changes that talk about this being optional from the other repo.